### PR TITLE
Create health-resources-for-your-office.md

### PR DESCRIPTION
**Pull request by monicazng**

### CODE REVIEW

File: src/perks/health-resources-for-your-home-office.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+### 🏃‍♂️ Move
+
+The number one rule is move! Everyone's body needs some stretching, standing up, and moving around every now and then. And it's easy to forget about moving when we're too focused on something. 
+
+Think about it as a system where when certain parts become more passive (your body), others overload to compensate (more mental effort to do things). Taking moving breaks helps sustain balance.
+
+Some people organize their days so that training, lunchtime, or picking up kids from school happen between focus work blocks. If you don't have a smartwatch, here are some tools that can remind you to have more often, shorter breaks:
+
+   * [Awareness](http://iamfutureproof.com/tools/awareness/)
```
Comment (Stefaniach) [inline src/perks/health-resources-for-your-home-office.md:16]: These 3 are a bit dated. Do we have any new ideas or links to add?

File: src/perks/health-resources-for-your-home-office.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+It's actually a pretty simple theory, and it's a matter of bringing consciousness to this area. With practice, it will eventually become intuitive and effortless to do.
+
+![alt text](https://www.healthandbalance.com.au/wp-content/uploads/2018/10/posture_desk.jpg)
+
+Experts recommend a posture where the neck, shoulder, and lower back are aligned with your monitor at eye level. Ideally, you'd switch from sitting to a standing desk, so you strike rules one and two all at once.
+
+Check out Workspace and equipment perks to get yourself a suitable desk and ergonomic chair setup, plus an external monitor if needed. 
+
+Most injuries are reversible, so if you experience pain or discomfort, seek advice. @pedro shared his experience on the matter:
```
Comment (Stefaniach) [inline src/perks/health-resources-for-your-home-office.md:35]: Pedro left so long ago.... maybe time to get new testimonials, or drop the testimonial idea entirely?

**Review by Stefaniach — APPROVED** [review]

File: src/perks/health-resources-for-your-home-office.md (hunk 1 above)
Comment (PabloMoralesMX) [reply src/perks/health-resources-for-your-home-office.md:16]: Samsung Health for Android: https://play.google.com/store/apps/details?id=com.sec.android.app.shealth&hl=en_US&gl=US

For IOS, Ergonomics App: https://apps.apple.com/app/ergonomics/id547689680

File: src/perks/health-resources-for-your-home-office.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+Buying a good monitor. Two things you can check: a) that there is no extra blue light in the LED, and b) that there is no PWM (Pulse Width Modulation), which is used as a cheap way to reduce brightness. PWM is turning on and off of the screen very fast instead of actually reducing the light. 
+
+Got any other tips and tricks for filtering blue light? Create a PR!
+
+
+### 🔆 Headspace: manage stress and unwind after work
+
+We've heard that a few folks would appreciate help managing stress, so we've secured some accounts with Headspace. Headspace is one of the leaders in this space, an app that focuses on mindfulness and meditation and has a range of tailored workouts for the mind. There's [research](https://www.headspace.com/science/meditation-research) that their app can positively impact stress, focus, mood, and work. 
+
+To use one of the subscriptions, signup with your @status.im email [here](https://work.headspace.com/status/join).
```
Comment (PabloMoralesMX) [inline src/perks/health-resources-for-your-home-office.md:60]: Just a headups, there´s no more space for signing in.  "Sorry, we’re all out of room. This program is currently full. Please contact your team's admin if you have any questions."

**Pull request merged**